### ISSUE TITLE
[docdb] ASAN leak in MetricsTest::DoLagTest

### ISSUE THREAD

**Issue by bmatican** [yugabyte/yugabyte-db#5762]
```
==16494==ERROR: LeakSanitizer: detected memory leaks

Direct leak of 48 byte(s) in 1 object(s) allocated from:
    #0 0x546248 in operator new(unsigned long) /opt/yb-build/thirdparty/yugabyte-db-thirdparty-v20200829090443-f431681041-centos/src/llvm-7.1.0.src/projects/compiler-rt/lib/asan/asan_new_delete.cc:106
    #1 0x55ba8e in void yb::MetricsTest::DoLagTest<yb::MillisLag>(yb::MillisLagPrototype const&) /nfusr/centos-gcp-cloud/jenkins-worker-24c/jenkins/jenkins-github-yugabyte-db-centos-master-clang-asan-1286/build/asan-clang-dynamic-ninja/../../src/yb/util/metrics-test.cc:71:16
```

**Comment by hulien22** [comment]
Didn't see that this github issue existed until I finished writing my diff @bmatican, but this should be fixed in e5494f1:
https://github.com/yugabyte/yugabyte-db/blob/685d448cf1e573305835049cda3b2c53176c009c/src/yb/util/metrics-test.cc#L71

**Comment by bmatican** [comment]
Oh, awesome @hulien22 , no worries, thanks for fixing this!
